src/views/pacbio/PacbioPoolCreate.vue

```diff
@@ -98,8 +84,19 @@ export default {
       'findPacbioPlate',
       'findPacbioTube',
     ]),
+    async fetchPoolsData() {
+      this.$store.commit('traction/pacbio/poolCreate/clearPoolData')
+      await this.fetchPacbioTagSets().then(this.alertOnFail)
+
+      // We should come up with a better solution to identify 'new' pools
```

Comment:
Could we use a type?

Reply:
That was an old comment I copied over, which I think i wrote, but I think this pattern is actually fine. But there is an issue here because if we use a random string it will try and look for a pool with that string, as we are only checking its not 'new'. Doesn't seem very safe. I don't have time to look before my leave but happy for you to have a go if you want, if not will look in the new year.